distro/README.md

```diff
@@ -0,0 +1,72 @@
+# Package `github.com/signalfx/splunk-otel-go/distro`
+
+This package provides a Splunk distribution of the OpenTelemetry Go SDK. It is
+designed to provide an SDK properly configured to be used with the Splunk
+platform out-of-the-box.
+
+## Getting Started
+
+The main entry point for the package is the [`Run`][] function. Use this
+function to create an SDK that is ready to be used with OpenTelemetry and
+forward all telemetry to Splunk. See [`example_test.go`](./example_test.go) for
+a complete example.
+
+## Configuration
+
+The [`SDK`][] is configured with the following options.
+
+| Option Name | Default Value | Environment Variable |
+| ---| --- | --- |
+| `WithAccessToken` | `""` | `SPLUNK_ACCESS_TOKEN` |
+| `WithEndpoint` | `"localhost:4317"`(1) | none |
+| `WithPropagator` | `tracecontext,baggage` | `OTEL_PROPAGATORS` |
+
+(1): The default value depends on the exporter used. See the
+[`WithEndpoint`](#withendpoint) section for more details.
+
+Environment variable can be used to set related option values, but the value
+set in code will take precedence. This is the same behavior the default
+OpenTelemetry SDK has.
+
+The following sections contain specific information for each option.
+
+### `WithAccessToken`
+
+`WithAccessToken` configures the authentication token used to authenticate
+telemetry delivery requests to a Splunk back-end.
+
+- Default value: empty (i.e. `""`)
+- Environment variable: `SPLUNK_ACCESS_TOKEN`
+
+### `WithEndpoint`
+
+`WithEndpoint` configures the Splunk endpoint that telemetry is sent to.
+
+- Default value: depends on the exporter used.
+  - For the `otlp` over gRPC exporter: `"localhost:4317"`
+  - For the `jaeger-thrift-splunk` exporter: `"http://127.0.0.1:9080/v1/trace"`
```

Comment:
how about documenting the env vars that can be used to set these values?

Reply:
Yeah, it looks like I included it in the follow on PR (https://github.com/signalfx/splunk-otel-go/pull/300/files#diff-f5ff2011d061002f61a86d92d59293383700444948d773a18a76625dd5986030). I'll move those changes forward.